src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1467,18 +1470,15 @@ namespace winrt::TerminalApp::implementation
 
     // Method Description:
     // - Split the focused pane either horizontally or vertically, and place the
-    //   given TermControl into the newly created pane.
+    //   given pane accordingly in the tree
     // Arguments:
+    // - newPane: the pane to add to our tree of panes
     // - splitDirection: one value from the TerminalApp::SplitDirection enum, indicating how the
     //   new pane should be split from its parent.
-    // - splitMode: value from TerminalApp::SplitType enum, indicating the profile to be used in the newly split pane.
-    // - newTerminalArgs: An object that may contain a blob of parameters to
-    //   control which profile is created and with possible other
-    //   configurations. See CascadiaSettings::BuildSettings for more details.
-    void TerminalPage::_SplitPane(const SplitDirection splitDirection,
-                                  const SplitType splitMode,
-                                  const float splitSize,
-                                  const NewTerminalArgs& newTerminalArgs)
+    // - splitSize: the size of the split
+    void TerminalPage::_SplitPane(std::shared_ptr<Pane> newPane,
```

Comment:
nit: the rest of the functions have the `newPane` as the last argument, it would be nice if this was consistent with them.

Reply:
Ah we can't here because there are default arguments that need to be the last few arguments (and the `newPane` is not a default argument)

Reply:
I think that the use in `SplitTab` is the only case where it actually uses the default parameters, since things are getting changed anyways I think it wouldn't be too much harm to make the `splitSize` parameter non-optional

Reply:
Fair enough, made the change!